src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -3863,6 +3863,9 @@ class SetStatementSegment(BaseSegment):
             Sequence(
                 "TIME", "ZONE", OneOf(Ref("QuotedLiteralSegment"), "LOCAL", "DEFAULT")
             ),
+            Sequence(
+                "SCHEMA", OneOf(Ref("QuotedLiteralSegment"))
```

Comment:
Unless you're adding something else here (perhaps an unquoted version?), then you shouldn't need the `OneOf` grammar, you can probably just do: `Sequence("SCHEMA", Ref("QuotedLiteralSegment"))`

Reply:
thanks, ive made this change